web_src/less/themes/theme-arc-green.less

```
@@ -1610,3 +1610,11 @@ footer .container .links > * {
 .repository.release #release-list > li .detail .dot {
     background-color: #888;
 }
+
+.signin {
+    .oauth2 {
+        img.openidConnect {
+                background-color: #fff;
+        }
+    }
+}
```

```suggestion
.signin .oauth2 img.openidConnect {
    background-color: #fff;
}
```
I think this style is more simple, and the result is same with what you want

---

Yes, no need to get fancy with Less unless it makes sense.

---

Thank you, I changed this. Below I added some pictures